### DISCUSSION TITLE
Outreachy Get Started

### DISCUSSION THREAD

**Discussion by NgweBecky** [camicroscope/caMicroscope#698] [General]
Hi everyone, I hope this beginner friendly guide helps you.
This was provided by our mentor @birm 
"I've put a list of some possible good contributor tasks here (https://github.com/camicroscope/GSOC/blob/master/ContributorTasks.md) "

Concerning any suggestions on contributing to the projects documentation, use this link:  http://camicroscope.org/docs/

**Comment by Kb-Jr** [comment]
Thank you very much @NgweBecky

**Reply by NgweBecky** [reply]
you are welcome

**Reply by nikky00** [reply]
Hi @NgweBecky ,i hope this finds you well
i would like to know if there is a discussion channel like slack/teams for this project asides here?Cos i'm a bit stuck and i do not know where to begin.

**Reply by birm** [reply]
We technically have a slack channel, but we discourage use of it since we're not paying for it, and the threads will eventually timeout and be list. What do you want to discuss? Can your questions fit into a discussion here?

**Reply by rchaurase** [reply]
Hello birm,

Apologies for any inconvenience caused. I have a question regarding the viewer repository. For every issue, is it necessary to create a separate pull request, or can multiple issues be addressed and merged automatically with a single pull request? I have already raised two pull requests, but I am currently unable to create a new pull request for resolving another issue.

Your guidance on this matter would be greatly appreciated.

Thank you.

**Comment by NgweBecky** [comment]
hi @nikky00 , to the best of my knowledge there is no other communication channel provided by our mentors. Can I know where you are stuck? I could be of help :)

**Comment by rchaurase** [comment]
Much appreciation, @NgweBecky. I have reviewed the ContributorTask file; the instructions are excellent and incredibly useful in organising my thoughts for the project.
I hope to produce something worthwhile and high-caliber with this project that I have started working on.

**Reply by NgweBecky** [reply]
you are welcome :)

**Comment by ssemanich** [comment]
hi everybody am new to the project and am happy to be here

**Reply by nwanduka** [reply]
You're welcome @ssemanich

**Comment by rimjhimittal** [comment]
Hello everyone,

I'm excited to join the Outreachy contribution phase with caMicroscope for the May to August 2024 internship cycle. I'm looking forward to contributing to the enhancement of biomedical image analysis and making the platform more accessible to users.

I'd love to hear your thoughts on key features or improvements that could benefit caMicroscope, especially regarding user accessibility and interface enhancements. Your insights will be incredibly helpful as I begin my contributions.

Looking forward to engaging with you all and learning from your experiences!

Best,
Rimjhim

P.S. A special shoutout to my mentors, Birm and Jasox. I'm eager to learn from your expertise and contribute to the caMicroscope community.

**Reply by DIANAAJAMBO** [reply]
Hello @rimjhimittal , you are very welcome. @birm provided this for us to get a head start with contributing. Please click on the link below to access the tasks. 
https://github.com/camicroscope/GSOC/blob/master/ContributorTasks.md

**Comment by Dorcas-BD** [comment]
Hello @birm, I'm Dorcas Bamisile, an outreachy applicant and am excited to contribute to caMicroscope community.

I will get started with the contribution tasks. Looking forward to learning a lot and making an impact on the community

Thank you.

**Reply by harshpreet14** [reply]
Welcome @Dorcas-BD

**Comment by olgangala2** [comment]
Hi @birm  I am called Ngala Olga and i am new to open source. I will like to work on ui/x and programmingg sector shown in the contributor task. I wish to ask how i can go about the instructions given. do i fork the repository and get started or there's an issue for it where where i can opt to work on

**Reply by olgangala2** [reply]
I am an outreachy applicant as well
